### PR TITLE
Add skeleton pages.

### PR DESCRIPTION
**Pull request by sgomes**
Simple, nearly-empty initial pages. Some hacky navigation for getting around for now, too.

### CODE REVIEW

File: src/server/js/routes/listing.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+
+import {Router} from 'express';
+const router = new Router();
```
Comment (devnook) [inline src/server/js/routes/listing.js]: I think we should not be creating a new router for every single route. Maybe we can put all of them in index.js?

**Comment by samdutton** [comment]
I'm not knowledgeable about Express, but LGTM.

One thing: we should remember to use semantic HTML elements. Some of these may be relevant: 

```
<article>
<aside>
<details>
<figcaption>
<figure>
<footer>
<header>
<main>
<nav>
<section>
<summary>
```

File: src/server/js/routes/listing.js (hunk 1 above)
Comment (rowan-m) [reply src/server/js/routes/listing.js]: Convention seems to be:
```javascript
import {Router} from 'express';
const router = Router();
```

e.g.
https://www.terlici.com/2014/09/29/express-router.html
https://github.com/searsaw/express-routing-example/blob/master/routes/cars/index.js

Comment (rowan-m) [reply src/server/js/routes/listing.js]: Ignore that comment, I don't know what I'm talking about.

My actual comment is that I feel each of these routing configurations will get larger as we do the actual implementation. I think it's prudent to separate them now even though they're trivial as we're just going to add more anyway.

Comment (sgomes) [reply src/server/js/routes/listing.js]: Keeping things separate for now, then.

Comment (devnook) [reply src/server/js/routes/listing.js]: Definitively we should keep handlers in separate files. But should we have a new router for each of them?

Comment (devnook) [reply src/server/js/routes/listing.js]: Ok. I still feel it's superfluous, but let's see how it goes.

**Pull request merged**